templates/AddNewCoreLibrary.ps1

```diff
@@ -0,0 +1,90 @@
+<#
+    .SYNOPSIS
+    Adds the class library and NuGet projects necessary to support a new core library
```

Comment:
Good thing also. Should I do it on the contributor.md document or on the CLI output also?